_includes/neurips/2019/presenters.nips.html

```diff
@@ -0,0 +1,146 @@
+<!-- Speakers Grid Section -->
+<section id="speakers" class="bg-light-gray">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Speakers</h2>
+                <h3 class="section-subheading text-muted"></h3>
+            </div>
+        </div>
+        <div class="grid">
+        {% assign featuredItems = site.tags.oral | sort: "featuredOrder" %} 
+        {% for post in featuredItems %}
+            <div class="cell speakers-item">
+                <a href="{{post.website}}" target="_blank" class="speakers-link">
+                    <div class="speakers-hover">
+                        <div class="speakers-hover-content">
+                            <p>{{post.topic}}</p>
+                            <i class="fa fa-plus fa-3x"></i>
+                        </div>
+                    </div>
+                    <img src="/assets/img/NIPS/Speakers/{{ post.img }}" class="responsive-image img-centered desaturate" alt="">
```

Comment:
This source reference string will need to be updated to include the year of this workshop for the presenter photos once they are added. See the current file structure for reference.